RELEASE.md

```diff
@@ -1,5 +1,5 @@
 # Releases #
-## In Progress Work ##
+## Release 2.4.2 (2017-09-25) ##
```

Comment:
I don't remember what versioning scheme api-checker follows... since a new configuration option was introduced, would this be a third-digit change?

Reply:
Let's ask @RackerWilliams .

Reply:
Latest word is to go ahead and bump it.
I'll change it from v2.4.2 to v2.5.0.